recipe/meta.yaml

```diff
@@ -24,13 +24,12 @@ requirements:
     - setuptools >=64
     - setuptools-scm >=6.4
   run:
-    # - exceptiongroup  # >=3.6.2
+    - exceptiongroup
```

Comment:
Without a selector, this will still get pulled in python 3.11 environments where it is unnecessary. However, since the alternative of doing architectured builds would drop 3.7 support (which this package advertises), I'm inclined to do this so that the package can remain `noarch`.

Reply:
there _is_ a way to do it, with icky `build_number` math. 

however, as one has to explicitly [_look for_](https://github.com/agronholm/anyio/blob/7ef43a6df150c93d2007f8f7f52e6901e37b16ae/src/anyio/_core/_sockets.py#L33) `exceptiongroup` on py<3.11, i don't think it has much of a chance of breaking anything... i wouldn't sweat an extra dependency.